tapdb/sqlc/migrations/000013_universe_fed_proof_sync_log.up.sql

```diff
@@ -0,0 +1,36 @@
+-- This table stores the log of federation universe proof sync attempts. Rows
+-- in this table are specific to a given proof leaf, server, and sync direction.
+CREATE TABLE IF NOT EXISTS federation_proof_sync_log (
+    id BIGINT PRIMARY KEY,
+
+    -- The status of the proof sync attempt.
+    status TEXT NOT NULL CHECK(status IN ('pending', 'complete')),
+
+    -- The timestamp of when the log entry for the associated proof was last
+    -- updated.
+    timestamp TIMESTAMP NOT NULL,
```

Comment:
Wondering if this should be a BIGINT and store Unix time like we did for universe events:

https://github.com/lightninglabs/taproot-assets/pull/396#discussion_r1263860576

Though I see we also have queries with comparisons to values using this `TIMESTAMP` type already?

Reply:
I think it's okay as long as the comparison in the queries is just greater than or less than (and not some fancy calculations), since in the end, `TIMESTAMP` in SQLite is just a string, formatted on the Golang side. So it works for ordering and filtering but parsing it in a SQL query is near impossible to get done for both SQLite and Postgres. So if we need to do any calculation (e.g. creating time buckets or things like that), we should use a unix timestamp.